integrationtest/mapper/orm/src/test/java/org/hibernate/search/integrationtest/mapper/orm/mapping/definition/AnnotationMappingDiscoveryIT.java

```diff
@@ -103,21 +105,7 @@ public void configure(HibernateOrmMappingConfigurationContext context) {
 
 	@Test
 	public void discoveryDisabled() {
-		backendMock.expectSchema( IndexedEntity.INDEX, b -> b
```

Comment:
You're completely changing the purpose of this test. Please don't do that.

This test is about checking that type annotation mapping discovery is effectively disabled. We used to check that by asserting that no mapping was contributed for the two `@IndexedEmbedded`. We can no longer do that because of the check you added.

One solution would be to add a programmatic mapping for the embedded types that adds some fields to these types, different from the ones configured through annotations. We would check that the programatically defined fields are added, but not the ones defined through annotations.

Reply:
Good idea. Let's do that!